docs/en/quickstart/data_import_guide.md

```diff
@@ -0,0 +1,42 @@
+# Data Import Quickstart
+ 
+There are 2 versions of OpenMLDB: the standalone version and cluster version.
+- For standalone version, datasets are all saved in the memory. Only [`LOAD DATA`](../reference/sql/dml/LOAD_DATA_STATEMENT.md) can be used to import data in this mode.
+- For cluster version, datasets can be saved offline or online. Offline and online ends don't share the data.
```

Comment:
- For the cluster version, datasets are stored separately in the offline and online storage engines.